modules/ROOT/pages/debugging-kernel-crashes.adoc

```diff
@@ -0,0 +1,32 @@
+= Enabling kdump to debug kernel crashes
```

Comment:
WDYT about rewording this as "Debugging kernel crashes using kdump" so that the title matches the title of the menu item more closely?